ajax/libs/lightservice/package.json

```diff
@@ -0,0 +1,28 @@
+{
+   "version": "9.1.0",
+   "filename": "lightservice.min.js",
+   "name": "lightservice",
+   "homepage": "https://github.com/contactsamie/LightService",
```

Comment:
Hello @contactsamie , we will use git repo as default homepage. Please remove this field. Thanks!